src/main/resources/application.conf

```diff
@@ -269,16 +269,16 @@ openolitor {
     # Project specific akka persistence configuration
     akka-persistence-sql-async {
       url = "jdbc:mysql://localhost:3306/csa1"
-      user = "dbusercsa1"
-      password = "thisDefaultUserPasswordMustBeChanedInProduction"
+      user = "root"
+      password = "thedefaultdbrootpasswordneedstobechanged"
```

Comment:
do you need to replace this?? I use this one in my local env

Reply:
no, this was a mistake! I revert it.